broker.go

```diff
@@ -175,17 +181,18 @@ type BrokerConf struct {
 
 func NewBrokerConf(clientID string) BrokerConf {
 	return BrokerConf{
-		ClientID:               clientID,
-		DialTimeout:            10 * time.Second,
-		DialRetryLimit:         10,
-		DialRetryWait:          500 * time.Millisecond,
-		AllowTopicCreation:     false,
-		LeaderRetryLimit:       10,
-		LeaderRetryWait:        500 * time.Millisecond,
-		MetadataRefreshTimeout: 30 * time.Second,
-		ConnectionLimit:        10,
-		IdleConnectionLimit:    5,
-		IdleConnectionWait:     200 * time.Millisecond,
+		ClientID:                 clientID,
+		DialTimeout:              10 * time.Second,
+		DialRetryLimit:           10,
+		DialRetryWait:            500 * time.Millisecond,
+		AllowTopicCreation:       false,
+		LeaderRetryLimit:         10,
+		LeaderRetryWait:          500 * time.Millisecond,
+		MetadataRefreshTimeout:   30 * time.Second,
+		MetadataRefreshFrequency: 0,
```

Comment:
I really don't have a good idea of what is a reasonable default, since it is pretty expensive. I figured if anybody else is using this client we can spare them a cluster meltdown by making them find the feature rather than surprising them.